app/containers/Header/ProfileDropdown.jsx

```diff
@@ -103,146 +103,154 @@ const B = ({ profileInfo, onClick, isMenuVisible, isMobileVersion }) => (
+          />
+          <FormattedMessage id={messages.profile.id} />
+        </Link>
+      ),
+    },
+    {
+      value: 2,
+      render: (
+        <Link to={routes.userCommunities(user)}>
+          <CommunitiesIcon
+            className="mr-2"
+            size={[18, 18]}
+            stroke={colors.linkColor || TEXT_PRIMARY}
+          />
+          <FormattedMessage id={messages.myCommunities.id} />
+        </Link>
+      ),
+    },
+    {
+      value: 3,
+      render: (
+        <Link
+          to={routes.userQuestions(user)}
+          disabled={!profileInfo.postCount}
+          tabIndex={!profileInfo.postCount ? '-1' : undefined}
+        >
+          <PostIcon
+            className="mr-2"
+            stroke={colors.linkColor || TEXT_PRIMARY}
+          />
+          <FormattedMessage id={messages.posts.id} />
+        </Link>
+      ),
+    },
+    {
+      value: 4,
+      render: (
+        <Link to={routes.userNotifications(user)}>
+          <NotificationsIcon
+            className="mr-2"
+            stroke={colors.linkColor || TEXT_PRIMARY}
+            fill={colors.linkColor || TEXT_PRIMARY}
+          />
+          <FormattedMessage id={messages.notifications.id} />
+        </Link>
+      ),
+    },
+    {
+      value: 5,
+      render: (
+        <Link to={routes.userNFTs(user)}>
+          <NFTIcon className="mr-2" stroke={colors.linkColor || TEXT_PRIMARY} />
+          <FormattedMessage id={messages.NFTs.id} />
+        </Link>
+      ),
+    },
+    isModerator && {
+      value: 6,
+      render: (
+        <Link to={routes.userModeration(user)}>
+          <ModerationIcon
+            size={[18, 18]}
+            className="mr-2"
+            stroke={colors.linkColor || TEXT_PRIMARY}
+            fill={colors.linkColor || TEXT_PRIMARY}
+          />
+          <FormattedMessage id={messages.moderation.id} />
+        </Link>
+      ),
+    },
+    {
+      value: 7,
+      render: (
+        <Link to={routes.userSettings(user)}>
+          <SettingsIcon
+            className="mr-2"
+            stroke={colors.linkColor || TEXT_PRIMARY}
+          />
+          <FormattedMessage id={messages.settings.id} />
+        </Link>
+      ),
+    },
+    {
+      value: 8,
+      render: (
+        <Logout>
+          <LogOutIcon
+            className="mr-2"
+            stroke={colors.linkColor || TEXT_PRIMARY}
+          />
+          <FormattedMessage id={messages.logout.id} />
+        </Logout>
+      ),
+    },
+  ];
+};
 
 const ProfileDropdown = ({ profileInfo }) => (
   <Dropdown
-    isArrowed
-    className="d-none d-md-flex"
     id="profile_dropdown_id"
-    button={<Button profileInfo={profileInfo} />}
-    menu={
-      <Menu
-        profileInfo={profileInfo}
-        questionsLength={profileInfo.postCount}
-        questionsWithUserAnswersLength={profileInfo.answersGiven}
-      />
-    }
+    options={getOptions(profileInfo).filter((option) => !!option)}
```

Comment:
add .filter((option) => !!option) to func getOptions

Reply:
done